modules/core/include/opencv2/core/matx.hpp

```diff
@@ -595,6 +595,7 @@ Matx<_Tp,m,n>::Matx(_Tp v0, _Tp v1, _Tp v2, _Tp v3, _Tp v4, _Tp v5, _Tp v6, _Tp
     val[4] = v4; val[5] = v5; val[6] = v6; val[7] = v7;
     val[8] = v8; val[9] = v9; val[10] = v10; val[11] = v11;
     val[12] = v12; val[13] = v13;
+    for (int i = 14; i < channels; i++) val[i] = _Tp(0);
```

Comment:
This line has already ensured that the number of channels is exactly 14.

Probably it might be changed to >= 14 to match the error message "at least 14 elements".